docs/features.rst

```diff
@@ -1,3 +1,134 @@
 Feature extraction
 ==================
 
+.. caution::
+    Lhotse feature extraction is still very much in the works. We will extend it to support custom feature extractors.
+
+Feature extraction in Lhotse is currently based exclusively on the `Torchaudio`_ library.
+We support spectrograms, log-Mel energies (*fbank*) and MFCCs.
+*Fbank* are the default features, and as of now the only ones supported in feature-space cut mixing.
+
+We are striving for a simple relation between the audio duration, the number of frames,
+and the frame shift.
+You only need to know two of those values to compute the third one, regardless of the frame length.
+This is equivalent of having Kaldi's _snip_edges_ parameter set to False.
+
+
+Storing features
+****************
+
+Features in Lhotse are stored as numpy matrices with shape ``(num_frames, num_features)``.
+By default, we use `lilcom`_ for lossy compression and reduce the size on the disk approximately by half.
```

Comment:
confused by underscore

Reply:
i just noticed that we have energy_floor 0.0 and dither 0.0 as default configs.  Seems like a dangerous combination as 0-energy signal might lead to NaN's.

Reply:
... also this file when compiled says `© Copyright 2020, Kaldi development team Revision 01f48c8d.` ... should be Lhotse development team?

Reply:
Underscore means its a hyperlink; the actual link is at the bottom of the page ("bibliography style").

I'll fix the energy_floor at 0.1.

Kaldi/Lhotse dev team - it's your call. I was under the impression we're still considering K2/Lhotse a part of the Kaldi effort.

Reply:
Thanks!  Let's call it Lhotse dev team for now.  It's all part of next-gen Kaldi but it's not clear that that will actually have the name "Kaldi" (since the project will be mostly independent code-wise, it might lead to confusion.)

Reply:
BTW the underscore didn't seem to display correctly, it displayed as  just an underscore.

Reply:
Oh, I can see it now. Yeah I'll fix that, thanks.